src/views/Dashboard/components/TopTenTools/TopTenTools.js

```diff
@@ -21,7 +21,9 @@ const GET_TOOL_EVENT_COUNT = gql`
 `
 
 const getDataProp = data => {
+  const colors = colorbrewer.Paired[10]
```

Comment:
Just a suggestion for improving `getDataProp`.

```js
const getDataProp = data => {
  const colors = colorbrewer.Paired[10]

  return data.reduce((acc, event, i) => {
    const { date, object_id: objectId, count } = event

    const chartProp = { x: new Date(date), y: count }

    if (!acc[objectId]) {
      acc[objectId] = {
        data: [chartProp],
        color: colors[Object.keys(acc).length]
      }
    } else {
      acc[objectId].data.push(chartProp)
    }
    return acc
  }, {})
}
```

I think this is a bit cleaner an implementation.